index.js

```diff
@@ -1,20 +1,21 @@
 /* global Blob */
 
-var once = require('once')
-
-module.exports = function getBlob (stream, mimeType, cb) {
-  if (typeof mimeType === 'function') return getBlob(stream, null, mimeType)
-  cb = once(cb)
-  var chunks = []
-  stream
-    .on('data', function (chunk) {
+module.exports = function (stream, mimeType) {
+  return new Promise(function (resolve, reject) {
+    if (typeof mimeType === 'function') {
```

Comment:
Instead of checking that it's a function and erroring, we should just confirm that it's what we want -- a string.